### ISSUE TITLE
Polishing the Game

### ISSUE THREAD

**Issue by blankRiot96** [Bit-Sahil04/Isometric-Minesweeper#1]
Hey, I tried running your game, functionality is great, it works like a charm
But it doesn't have much Polish to it, would you mind if i added things like particle effects and screen shake?

**Comment by Bit-Sahil04** [comment]
Absolutely! Was planning on refactoring code and adding polish myself for a long time but didn't commit to the plan. Feel free to send in a PR.

**Comment by blankRiot96** [comment]
Oh thats great, im working on it!
